### DISCUSSION TITLE
Why does Oqtane need the preview edition of Visual Studio 2019?

### DISCUSSION THREAD

**Discussion by abukres** [oqtane/oqtane.framework#755] [General]
I don't like installing preview editions of VS. I know MS says it can be installed side by side with the stable editions but I came across when this wasn't true. Like when I uninstalled the preview, it messed up my stable one. Maybe this won't happen anymore but I prefer not to taken chances unless I have to.

I suggest Oqtane's GitHub page mentions exactly which preview version it needs and why. Maybe the readme was written for prior preview versions of VS whose official versions have been released already and this requirement is not up to date and not valid anymore. So does Oqtane require Visual Studio 2019 version 16.8 Preview 3? What does Oqtane require in there? Maybe I'll just wait till the official release comes out. 

I appreciate more detailed information be released to us so we can make better informed decisions.

**Comment by hishamco** [comment]
I will update the docs, but the reason if I'm not wrong at that time preview versions of VS 2019 supported ASP.NET Core and Blazor

**Reply by hishamco** [reply]
The PR is created https://github.com/oqtane/oqtane.framework/pull/756
